### ISSUE TITLE
About the pre-training 

### ISSUE THREAD

**Issue by kikyou123** [YU1ut/imprinted-weights#4]
It seems that you pre-train the model with all the training images, including the base and novel classes. I argue it is unfair to use the novel classes to train the model.

**Comment by YU1ut** [comment]
No. Only base classes are used in pre-training. 
https://github.com/YU1ut/imprinted-weights/blob/master/loader.py#L27
And the network only classifies 100 classes.
https://github.com/YU1ut/imprinted-weights/blob/master/models.py#L6

**Comment by blue-blue272** [comment]
Yes, I understand. Thanks for replying!

**Comment by blue-blue272** [comment]
I have another question. When you pretrain the model, you set the train loader as :
`  train_dataset = loader.ImageLoader(
        args.data,
        transforms.Compose([
            transforms.RandomResizedCrop(224),
            transforms.RandomHorizontalFlip(),
            transforms.ToTensor(),
            normalize,
        ]), train=True)`
I argue should you insert the `transforms.Resize(256),` before `  transforms.RandomResizedCrop(224),`
